release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -405,6 +405,12 @@ Previously, if one pod (`Pod X`) was deleted, and a second pod (`Pod Y`) was cre
 
 With this update, pods created using the macvlan CNI plugin, where the IP address management CNI plugin has assigned IPs, now send IPv6 unsolicited neighbor advertisements by default onto the network. This enhancement notifies the network fabric of the new pod's MAC address for a particular IP to refresh IPv6 neighbor caches.
 
+
+[id="ocp-4-15-configuring-whereabouts-IP-reconciler-schedule"]
+==== Configuring the Whereabouts IP reconciler schedule
+
+The Whereabouts reconciliation schedule was hard-coded to run once per day and could not be reconfigured. With this release, a `ConfigMap` has enabled the configuration of the whereabouts cron schedule. For more information, see xref:../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-configuring-whereabouts-ip-reconciler-schedule_configuring-additional-network[Configuring the Whereabouts IP reconciler schedule].
```

Comment:
As discussed offline, PTAL at my comments in the similar 4.13 RN PR, so we can hopefully get both the style (and links, eventually) synced up across the versions:

https://github.com/openshift/openshift-docs/pull/71260#discussion_r1482033816